README.md

```diff
@@ -7,13 +7,12 @@
 > [!NOTE]  
 > This product is currently in **Beta**. We're excited to share it with you and welcome your feedback to help us improve. While the core features are ready for use, you may encounter minor issues as we continue to refine the experience. Please share your thoughts and suggestions in the [~user-feedback](https://community.mattermost.com/core/channels/user-feedback) channel or submit an issue on [GitHub](https://github.com/mattermost/mattermost-plugin-msteams-devsecops/issues).
 
-This repository provides the foundation for multiple Microsoft app offerings that integrate with the Mattermost platform. 
-
+This repository provides the foundation for multiple Microsoft app offerings that integrate with the Mattermost platform, including:
 
 - **Mattermost Mission Collaboration for Microsoft**: Extend Enterprise communications to the Tactical Edge by accessing Mattermost from your Microsoft Office365, Teams, and Outlook user experiences. 
-- **Community Mattermost for Microsoft 365 & MS Teams**: Free offering to explore Mattermost capabilities and meet with fellow end users, customers and evaluators, along with Mattermost staff. 
+- **Community Mattermost for Microsoft 365 & MS Teams**: Free offering to explore Mattermost capabilities and meet with fellow end users, customers and evaluators, along with Mattermost staff. [Learn more](#community-mattermost-for-microsoft-365-and-ms-teams)
```

Comment:
Community can be removed here as well.  Plus we won't need the "provides the foundation for multiple Microsoft app offerings" line either as this repo is focused on one product.  

I think we've gutted this README to the point that it no longer describes what this product is at a glance.  In a new PR I'll add a separate line near the top that tells the simple story that this product embeds Mattermost in Teams.

Reply:
Thanks @wiggin77! My latest commit removes these unnecessary references.